### PR TITLE
TYPOS

### PR DESCRIPTION
**Pull request by CanDursun**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -59,19 +59,19 @@ sudo apt install libusb-1.0-0-dev
...
+On FreeBSD, you will need to install libiconv. This is done by running
 the following:
 ```sh
 pkg_add -r libiconv
 ```
 
 ### Mac:
 
-On Mac make sure you have XCode installed and its Command Line Tools.
+make sure you have XCode installed and its Command Line Tools.
```
Comment (Youw) [inline BUILD.md:70]:
```suggestion
Make sure you have XCode installed and its Command Line Tools.
```

File: BUILD.md [hunk 2]
```diff
@@ -14,21 +14,21 @@
...
 It can be done in several different ways:
 - using [CMake](BUILD.cmake.md);
 - using [Autotools](BUILD.autotools.md) (deprecated);
 - using [manual makefiles](#building-the-manual-way-on-unix-platforms);
 - using `Meson` (requires CMake);
 
-**Autotools** build system is historically first mature build system for
-HIDAPI. Most common usage of it is in its separate README: [BUILD.autotools.md](BUILD.autotools.md).<br/>
+**Autotools** build system is historically the first mature build system for
+HIDAPI. The most common usage of it is in its separate README: [BUILD.autotools.MD](BUILD.autotools.md).<br/>
```
Comment (Youw) [inline BUILD.md:26]: The filename has a lower case axtension.
```suggestion
HIDAPI. The most common usage of it is in its separate README: [BUILD.autotools.md](BUILD.autotools.md).<br/>
```

**Review by Youw — APPROVED** [review]
Thanks, I was hoping someone would check it out.

**Comment by CanDursun** [comment]
You are welcome. That was my first pull request. Thanks to you!

**Comment by Youw** [comment]
> That was my first pull request.

I think you are yet to learn how pull requests work :)

**Pull request merged**